.ci/gitlab/.gitlab-ci.yml

```diff
@@ -291,7 +291,7 @@ e4s-neoverse-v2-build:
 
 e4s-rocm-external-generate:
   extends: [ ".e4s-rocm-external", ".generate-x86_64"]
-  image: ghcr.io/spack/spack/ubuntu22.04-runner-amd64-gcc-11.4-rocm6.2.1:2024.10.08
+  image: ecpe4s/ubuntu24.04-runner-amd64-gcc-13.3-rocm-6.4.3:2025.09.22
```

Comment:
My only concern here is rate limiting issues pulling from docker vs ghcr. I believe that is why we moved them previously. Do we know if those have been resolved?

Reply:
Absolutely, you are right. This is my first time using a newer version of rocm build in a long time, wanted to just get some ci results. We will update the PR with ghcr hosted container image if things pass here.

Reply:
Although, maybe that can't wait-- I am seeing some rate limits already have been hit. I thought the limits were higher. I will move the container to ghcr. Thanks for flagging.

Reply:
Update pushed!